### DISCUSSION TITLE
Eliminating the effect of the Earth's radius on the visible position of planets.

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2068] [General]
Stellarium allows to change the height up to 2000 meters below sea level.
This is the classic situation where altitude has an effect on the observed angle of celestial bodies, especially those close to the horizon.
When I want to observe the position of the planets and the effect of Kepler's laws on their orbit, I try to eliminate that effect.
Is it possible to calculate the position as if the observer is in the center of the earth.
(At the equator, for example, This is an altitude of -6,378,137 meters.)

**Comment by alex-w** [comment]
See "Configuration window" [F2] -> "Tools" tab -> "Topocentric coordinates"

**Comment by tzmosh** [comment]
Thanks. I really appreciate your activity in general for Stellarium.
